packages/layout/src/stack.tsx

```diff
@@ -77,7 +77,7 @@ export const Stack = React.forwardRef(function Stack(
 ) {
   const {
     direction = "column",
-    align = "flex-start",
+    align = "normal",
```

Comment:
I see what you mean. We definitely want to reduce breaking changes as much as we can.

I think the best thing to do in this case, is to completely remove the align prop's default value.

Reply:
No problemo 👌